src/vm/amd64/jithelpers_fast.S

```diff
@@ -10,6 +10,45 @@ LEAF_ENTRY JIT_PatchedCodeStart, _TEXT
         ret
 LEAF_END JIT_PatchedCodeStart, _TEXT
 
+
```

Comment:
Is there a reason for moving this function here? It seems to just complicate the diff of the change.